cves/kernel/CVE-2013-0349.yml

```diff
@@ -147,10 +156,22 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: | 
```

Comment:
Good summary, however are there any sources, other than Red Hat Security, that you used? Are there hints in some of the pull requests or previous mentions of this?

Reply:
I also looked at the GitHub commit, the Ubuntu security notice, the Bugzilla report, and the openwall communications, trying to find the original discoverer.  None of these had definitive information.